src/pages/RequestAccess/RequestAccess.js

```diff
@@ -32,8 +32,10 @@ const RequestAccess = ({
     label: p.display,
     value: p.code,
   }));
-  const hiTypesInitialStates = Object.assign(
-    ...requestHiTypes.map((hiType) => ({ [hiType.value]: false }))
+  var hiTypesInitialStates = Object.assign(
```

Comment:
Any specific reason of using `var` instead of `let`?

Reply:
I think we can use 'const' here as its value is never reassigned

Reply:
Yes.